test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -33,7 +31,7 @@ public void testPrepareSearchURL() {
             url = TwitterScraper.prepareSearchURL(query[i]);
 
             //compare urls with urls created
-            assertEquals(url, out_url[i]);
+            assertThat(out_url[i], is(url));
```

Comment:
http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/Matchers.html
Inside is matcher calls equals of the string. I think that equalTo is redudant for this case and do not this assert is more readable